config/environments/test.rb

```diff
@@ -50,4 +50,6 @@
 
   # Annotate rendered view with file names.
   # config.action_view.annotate_rendered_view_with_filenames = true
+
+  config.log_level = :warn
```

Comment:
do we need warn? Plus the test below is testing for an :info statement

Reply:
have checked and even when this is removed the default level of :debug does not result in the output coming out in the console when running the test. leaving it as :warn results in no output in the `log/test.log` however, which could be confusing for our future selves and other devs. I would remove it, and the development one two unless there is a reason why we want to change the defaults.

Reply:
👍 have removed onfig.log_level = :warn